src/bookshelf/extras.ts

```diff
@@ -11,6 +11,7 @@ export interface Model extends BModel<any> {
   id: any; // TODO IMPROVE REAL TYPE
   attributes: any;
   relations:  RelationsObject;
+  models: any;
```

Comment:
Yes I'll restore prepublish, and check out your other comment. It was
actually a co workers code and I didnt have time to review it and they were
blocked by it. I didn't mean to send you a pull request, I'm still be to
GitHub .😏

On Thu, Jul 28, 2016, 1:10 PM Manuel Pacheco <notifications@github.com>
wrote:

> In src/bookshelf/extras.ts
> <https://github.com/scoutforpets/jsonapi-mapper/pull/54#discussion_r72662269>
> :
>
> > @@ -11,6 +11,7 @@ export interface Model extends BModel<any> {
> >    id: any; // TODO IMPROVE REAL TYPE
> >    attributes: any;
> >    relations:  RelationsObject;
> > +  models: any;
>
> This shouldn't be here: a Model from Bookshelf doesn't have an internal
> models property (remember these extra interfaces exposes private properties
> because we use them to analyze the object)
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/scoutforpets/jsonapi-mapper/pull/54/files/aa522604db848ba4197589635bbc0aca7aa8dc91#r72662269>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAt-V-242bTXXzrOLpY6XmdGL2WFspWmks5qaOKUgaJpZM4JWhXD>
> .
>